app/src/components/submissionPage/SnapList.tsx

```diff
@@ -44,15 +38,16 @@ const SnapCardSpacer = styled.div`
 `;
 
 const SnapCardFooter = styled.p`
-    color: var(--text-muted);
+    color: var(--text-main);
     padding: 0;
     margin: 0;
 `;
 
 const SnapList: React.FunctionComponent<SnapListProps> = ({ snaps }) => {
-    function formatAts(ats: String[]): String {
-        return ats.map((s) => `@${s}`).join(" ");
-    }
+    // The regex matches something of the form [full name]
+    const fullNameRegex = /\[[A-Za-z0-9 _]*\]/g;
+    // The regex matches something of the form @[full name](username)
+    const tagRegex = /@\[[A-Za-z0-9 _]*\]\([A-Za-z0-9 _]*\)/g;
```

Comment:
We can simplify this. We don't really care what the full name and username is, so long as they conform to the general format: `@[name](username)`. Using `.+` instead of `A-Za-z0-9 _` should be suffient. We can then use [capture groups](https://www.regular-expressions.info/brackets.html) to identify the full name and username based on their position within the overall match.